demo/src/androidTest/java/com/novoda/movies/rateable/RateableMovieViewHolderTest.java

```diff
@@ -29,7 +29,7 @@
 import static android.support.test.espresso.matcher.ViewMatchers.withId;
 import static android.support.test.espresso.matcher.ViewMatchers.withText;
 import static com.novoda.espresso.ViewTestRule.underTest;
-import static com.novoda.movies.topten.RateableMovieViewModelFixtures.viewModel;
+import static com.novoda.movies.rateable.RateableMovieViewModelFixtures.viewModel;
```

Comment:
I truly forgot that I had these in a base branch already else I'd have avoided the package change 🙏